PendingReleaseNotes

```diff
@@ -226,6 +226,10 @@
   autoscaling, see :ref:`pg-autoscaler`.  Note that existing pools in
   upgraded clusters will still be set to ``warn`` by default.
 
+* The pool parameter ``target_size_ratio``, used by the pg autoscaler,
+  has changed meaning. It is now normalized across pools, rather than
+  specifying an absolute ratio. For details, see :ref:`pg-autoscaler`.
+
```

Comment:
i wonder if we should warn that an upgrade may result in moved data?  if operators are cautious/concerned, they should set their pools to 'warn' mode instead of 'on' before upgrading?

Reply:
added